MongoUsingMongoose/server.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+// Configuring the database
+const dbConfig = require('./config.js');
+// create express app
+const app = express();
+var http = require('http');
+var httpServer = http.createServer(app);
+// parse requests of content-type 
+app.use(bodyParser.urlencoded({ extended: true }))
+// parse requests of content-type - application/json
+app.use(bodyParser.json())
+//Reference of the router file
+const routerObj = require('./routes');
+app.use('/',routerObj);
+//connecting to the database !!!
+dbConfig.connect();
```

Comment:
server should not start if db connection is failed

Reply:
Done